### PR TITLE
support preempting task with bound status

### PR DESCRIPTION
**Pull request by Lily922**
fix issue: https://github.com/volcano-sh/volcano/issues/3207

### CODE REVIEW

File: pkg/scheduler/api/helpers.go [hunk 1]
```diff
@@ -60,6 +60,16 @@ func getTaskStatus(pod *v1.Pod) TaskStatus {
 	return Unknown
 }
 
+// PreemptibleStatus checks whether the task can be preempted
+func PreemptibleStatus(status TaskStatus) bool {
```
Comment (william-wang) [inline pkg/scheduler/api/helpers.go:64]: fix the typo from `PreemptibleStatus` to `PreemptableStatus`

Comment (Lily922) [reply pkg/scheduler/api/helpers.go]: Done

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**